2017/Cambridge-OMERO-API/index.html

```diff
@@ -0,0 +1,268 @@
+<!doctype html>
+
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+
+        <title></title>
+
+        <meta name="apple-mobile-web-app-capable" content="yes" />
+        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+        <link rel="stylesheet" href="reveal-js/css/reveal.min.css">
+        <link rel="stylesheet" href="reveal-js/css/theme/default.css" id="theme">
+
+        <!-- OME presentation overrides of the default theme -->
+        <link rel="stylesheet" href="css/ome.css">
+
+        <!-- For syntax highlighting -->
+        <link rel="stylesheet" href="reveal-js/lib/css/zenburn.css">
+
+        <!-- If the query includes 'print-pdf', use the PDF print sheet -->
+        <script>
+            document.write( '<link rel="stylesheet" href="reveal-js/css/print/' + ( window.location.search.match( /print-pdf/gi ) ? 'pdf' : 'paper' ) + '.css" type="text/css" media="print">' );
+        </script>
+
+    </head>
+
+    <body class="ome_theme">
+
+        <div class="reveal">
+
+            <!-- Any section element inside of this container is displayed as a slide -->
+            <div class="slides">
+
+                <!-- title page -->
+                <section>
+                    <h1>Image Processing with OMERO</h1>
+                    <p>
+                        Cambridge <br>
+                        8th December, 2017 <br>
+                        Will Moore, Balaji Ramalingam, Petr Walczysko
+                    </p>
+                </section>
+                
+                <section>
+                    <h2>Outline of the Day</h2>
+                    <ol>
+                        <li>Presentation
+                            <ul>
+                                <li>What is OMERO?</li>
+                                <li>OMERO model</li>
+                                <li>OMERO API
+                            </ul>
+                        </li>
+                        <li>ImageJ/Fiji as an OMERO client</li>
+                        <li>OMERO.scripts with SciPy
+                            <ul><li>OMERO.figure scripting with JSON</li></ul>
+                        </li>
+                        <li>Jupyter Notebooks
+                            <ul><li>Python/CellProfiler | R</li></ul>
```

Comment:
following discussion an OMEROHelloWorld has been added
so maybe better to have Python CP and R